src/components/jnlp-page/Hero-JNLP.astro

```diff
@@ -0,0 +1,67 @@
+---
+import LinkButton from "../LinkButton.astro";
+import { Image } from "astro:assets";
+import { DISCORD_URL } from "../../consts";
+import Icon from "astro-icon";
+import JNLPBox from "./assets/box-JNLP.png";
+import sparkleIcon from "./assets/sparkle.png";
+---
+
+<header
+	class="min-h-[calc(100vh-48px)] flex flex-col items-center bg-gradient-to-b to-[#e6275544] from-stone-950"
+>
+	<div class="grow"></div>
+
+	<div
+		class="w-full max-w-screen-xl px-4 sm:px-8 py-16 space-y-4 md:grid grid-cols-3"
+	>
+		<div class="flex items-center col-span-2">
+			<div>
+				<h1
+					class="text-white text-4xl md:text-5xl lg:text-6xl xl:text-7xl font-semibold"
+				>
+					Run
+					<span class="font-bold text-primary-500"> Java Web Start</span>
+					<br />apps in the browser
+				</h1>
+				<p class="text-xl mt-8 text-stone-400 leading-6">
+					Unleash the power of CheerpJ now available in a easy-to-use browser
+					extension. Run Java Web Start Applications, including Oracle
+					E-Business Suite and Oracle Forms Applications, without a Java Desktop
+					installation.
+				</p>
+				<div class="mt-8 flex flex-wrap gap-4">
+					<LinkButton
+						href="#ecosystem"
+						label="Add to Chrome"
+						type="primary"
+						iconRight="ri:chrome-fill"
+					/>
+					<LinkButton
+						href="#"
+						label="Add to Edge"
+						type="secondary"
```

Comment:
Having this secondary doesn't make sense to me.

Can we instead make a new component that picks the right LinkButton to use/highlight depending on what the user's browser is? Instead of user agent sniffing (to be deprecated), we can use [User Agent Client Hints](https://developer.chrome.com/docs/privacy-security/user-agent-client-hints) on Chromium derivatives like Chrome and Edge (that is, the API is not supported on Firefox at the moment, but we don't care).

Reply:
I tried implementing this and I found it tricky since window.navigator is not accessible from Astro,  only using <script> plus I need to find a way to successfully share variables from one side to the other. I am sure there is a way but I need to dig more.
I did a workaround for now and made another type of button that looks good too. I still want to implement this idea though.

Reply:
Ah yeah this is a bit tricky. Perhaps we make this a svelte component and `client:idle` it. We'll need to solve the function colouring issue of Svelte not being able to use Astro components though...

Reply:
OK, let's call this out-of-scope for this PR for complexity reasons. The current solution is passable. How about we make an Asana task out of this and tackle it later?